### DISCUSSION TITLE
Chinese transcript with smart formatting add spaces between each symbol.

### DISCUSSION THREAD

**Discussion by emilbryggare** [deepgram/community#1017] [General help]
Hi there! 

We are using the Nova 2 model with Smart Format enabled to generate transcripts. When transcribing into Chinese, each symbol is separated by a space in the formatted output. Is this intentional? It does not seem correct to us.

<img width="884" alt="image" src="https://github.com/user-attachments/assets/4ce734f9-905d-4aad-8f68-eb5c8bec3a81">

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Reply by emilbryggare** [reply]
The redirect_uri seem to be misconfigured for this
![CleanShot 2024-12-03 at 10 32 44@2x](https://github.com/user-attachments/assets/49f634af-e9a7-40a9-944e-498d55348111)

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Reply by emilbryggare** [reply]
Speech to text. We are using Node - but reproducible in the online playground.
request_id:"101a68db-e876-405d-858f-a29e331ba98d"
